src-cpp/HandleImpl.cpp

```diff
@@ -282,6 +282,25 @@ RdKafka::HandleImpl::resume (std::vector<RdKafka::TopicPartition*> &partitions)
   return static_cast<RdKafka::ErrorCode>(err);
 }
 
+RdKafka::Queue *
+RdKafka::HandleImpl::get_partition_queue (TopicPartition *part) {
+  if (part == NULL)
```

Comment:
These kinds of checks are not required, they just add noise.

It is a application side error to call most functions with invalid parameters (including NULL)